src/objects.rs

```diff
@@ -17,30 +17,33 @@ use std::ops::{Deref, DerefMut};
 /// tags](http://wiki.openstreetmap.org/wiki/Tags) for more
 /// information.
 #[derive(Debug, Default, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
-pub struct Tags(TagsImpl);
-/// FlatMap representing the key-value pairs of the tags
-pub type TagsImpl = ::flat_map::FlatMap<String, String>;
```

Comment:
You must let this type alias, else it's a breaking change.

Reply:
It doesn't change the API, so from my point of view it isn't one.

Reply:
All codes using your crate won't break with this change.

Reply:
It will if someone use something like `struct Data { tags: osmpbfreader::objects::TagsImpl }`

Reply:
Good point. Do you mind if we remove it then?

Reply:
I'd prefer to keep it, but no strong feeling on it (I doubt someone is using it).